### DISCUSSION TITLE
Can an individual have multiple paid accounts?

### DISCUSSION THREAD

**Discussion by justanesta** [community/community#138495] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello all,

I like many of you have had my free account flagged for reasons I'm still not certain about. I've of course submitted a help ticked but based on my research am not expecting a response anytime soon, if at all.

I still need to use GitHub and at this point am willing to create a new account. I know the GitHub ToS is very explicit that multiple _free_ personal accounts are not allowed. That said, since I am unable to delete my current flagged account would it be violating the ToS to create a second personal account that is _paid_ through GitHub Pro? 

Any guidance, particularly from GitHub staff would be greatly appreciated. I don't want to run afoul with the GitHub ToS but would really like to continue using the service if I am able to!

**Comment by ghost** [comment]
Hello

**Reply by JK-77** [reply]
yess

> Hello

nn

**Comment by JK-77** [comment]
hello
